src/components/TimeZoneSelector.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useEffect, useMemo, useState } from 'react';
+import { Autocomplete } from '@terraware/web-components';
+import strings from 'src/strings';
+import { getTimeZones } from 'src/api/timezones/timezones';
+
+export type TimeZoneDescription = {
+  id: string;
+  longName: string;
+};
+
+export type TimeZoneSelectorProps = {
+  onTimeZoneSelected: (tzSelected: TimeZoneDescription) => void;
+  selectedTimeZone?: string;
+};
+
+export default function TimeZoneSelector(props: TimeZoneSelectorProps): JSX.Element {
+  const { onTimeZoneSelected, selectedTimeZone } = props;
+  const [timeZones, setTimeZones] = useState<TimeZoneDescription[]>([]);
+
+  useEffect(() => {
+    const fetchTimeZones = async () => {
+      const timeZoneResponse = await getTimeZones();
+      if (!timeZoneResponse.error && timeZoneResponse.timeZones) {
+        setTimeZones(
+          timeZoneResponse.timeZones.sort((a, b) => {
+            if (a.longName > b.longName) {
+              return 1;
+            }
+            if (b.longName > a.longName) {
+              return -1;
+            }
+            return 0;
```

Comment:
You may be able to replace this with simply
```typescript
timeZoneResponse.timeZones.sort((a, b) => a.longName > b.longName)
```

Reply:
nope sorry, that's wrong, please ignore 😅

Reply:
this is what i was thinking of: `a.longName.compareTo(b.longName)`

Reply:
Oh, this looks nicer. Will test. Thanks @charlie-tf !

Reply:
actually it's `a.longName.localeCompare(b.longName)`